contracts/manifest.md

```diff
@@ -0,0 +1,21 @@
+NAIS manifest
+=============
+
+This file contains the configuration of your NAIS application. It must reside within the git repository together with your source code.
```

Comment:
Skriver det om til noe annet;
The NAIS manifest contains the configuration of your NAIS application. It can either be uploaded to our [Nexus](/dev-guide/nexus.md), or specify an [external manifest](dev-guide\
/naisd.md#external-or-non-default-manifest-address) with `manifestUrl` in the [deploy](/dev-guide/niasd-md#deploy)-request.